packages/ses/src/whitelist.js

```diff
@@ -1101,6 +1101,7 @@ export const whitelist = {
   DataView: {
     // Properties of the DataView Constructor
     '[[Proto]]': '%FunctionPrototype%',
+    BYTES_PER_ELEMENT: 'number', // Non std but undeletable on Safari.
```

Comment:
Does this reveal something about the platform that could be a thread to determinism? Not that we can do much about it..

Reply:
It falls within static implementation non-determinism https://github.com/tc39/proposal-ses#how-deterministic , just like sort order does. At some point when we have a fully deterministic specification, none of the browsers will implement it because of these sorts of static non-determinism. This static form of non-determinsm does not threaten security. It threatens only deterministic replication and replay, which we're never gonna try on browser JS engines anyway.